### PR TITLE
Rfc453 issue cred 2.0

### PR DESCRIPTION
**Pull request by sklump**
Plus fixes for bugs/repetitiveness found along the way

### CODE REVIEW

**Comment by ianco** [comment]
Looks good to me, I'll leave the official review for @andrewwhitehead however

File: aries_cloudagent/protocols/issue_credential/v2_0/routes.py [hunk 1]
```diff
@@ -0,0 +1,1343 @@
...
+        required=False,
+    )
+    dif = fields.Nested(
+        V20CredExRecordDIFSchema,
+        required=False,
+    )
+
+
+class V20CredExRecordListResultSchema(OpenAPISchema):
+    """Result schema for credential exchange record list uery."""
```
Comment (TimoGlastra) [inline aries_cloudagent/protocols/issue_credential/v2_0/routes.py:111]:
```suggestion
    """Result schema for credential exchange record list query."""
```

File: aries_cloudagent/ledger/indy.py [hunk 2]
```diff
@@ -1048,25 +1048,12 @@ async def accept_txn_author_agreement(
...
     async def get_latest_txn_author_acceptance(self) -> dict:
         """Look up the latest TAA acceptance."""
-        cache_key = (
-            TAA_ACCEPTED_RECORD_TYPE
-            + "::"
-            + self.wallet.opened.name
-            + "::"
-            + self.pool.name
-        )
+        cache_key = TAA_ACCEPTED_RECORD_TYPE + "::" + self.pool.name
```
[the remark this answers is not in the document]
Comment (sklump) [reply aries_cloudagent/ledger/indy.py:1056]: It must have come from a merge with master: it doesn't look like mine - I would always use an f-string here

File: aries_cloudagent/protocols/issue_credential/v2_0/routes.py (hunk 1 above)
Comment (sklump) [reply aries_cloudagent/protocols/issue_credential/v2_0/routes.py:111]: Are you volunteering to fix all the typos in all the comments? If so, you're a champ :-D

**Comment by sklump** [comment]
OMG do NOT use the "commit this suggestion" button: it leaves off the DCO and drops you into github jail

**Comment by sklump** [comment]
I can't fight the evil. New branch, new PR. All for the sake of a q.

**Pull request closed without merging**